ganache/deploy.js

```diff
@@ -3,7 +3,7 @@ const {
   ERC20,
   toSmartContractDecimals,
   Network
-} = require("bepro-js");
+} = require("@taikai/dappkit");
```

Comment:
I'd say this script is moot since we're introducing the same script on `scripts/deploy.js` ? can we remove this?

@clarkjoao @vhcsilva

Reply:
Yes for me

Reply:
@hvasconcelos remove this file please :)